app/controllers/api/database.php

```diff
@@ -783,12 +741,7 @@ function createAttribute(string $collectionId, Document $attribute, Response $re
     ->inject('audits')
     ->inject('usage')
     ->inject('events')
-    ->action(function ($collectionId, $key, $required, $default, $array, $response, $dbForProject, $database, $audits, $usage, $events) {
-        /** @var Appwrite\Utopia\Response $response */
-        /** @var Utopia\Database\Database $dbForProject*/
-        /** @var Appwrite\Event\Database $database */
-        /** @var Appwrite\Event\Audit $audits */
-        /** @var Appwrite\Stats\Stats $usage */
+    ->action(function (string $collectionId, string $key, bool $required, ?string $default, bool $array, Response $response, Database $dbForProject, EventDatabase $database, Event $audits, Stats $usage, Event $events) {
```

Comment:
Here as well, isn't $required nullable ? 
Please check in the other controllers as well.

Reply:
updated